GVFS/GVFS.Common/GitStatusCacheConfig.cs

```diff
@@ -86,16 +86,10 @@ private static bool TryGetFromGitConfig(GitProcess git, string configName, int d
             error = string.Empty;
 
             GitProcess.Result result = git.GetFromConfig(configName);

```

Comment:
Do you know if there are any unit tests or functional tests covering this code path?  We should be able to use the `MockGitProcess` if not.